fe/fe-core/src/main/java/com/starrocks/statistic/AnalyzeStatus.java

```diff
@@ -163,12 +174,12 @@ public ShowResultSet toShowResult() {
 
         String msgType;
         String msgText;
-        if (status.equals(StatsConstants.ScheduleStatus.FINISH)) {
-            msgType = "status";
-            msgText = "OK";
-        } else {
+        if (status.equals(StatsConstants.ScheduleStatus.FAILED)) {
             msgType = "error";
             msgText = reason;
+        } else {
+            msgType = "status";
+            msgText = "OK";
```

Comment:
both of PENDING/RUNING/FINISH return ok？

Reply:
Yes, here is mainly compatible with the return value of mysql